code/modules/jobs/job_types/ncr.dm

```diff
@@ -95,341 +105,350 @@ Captain
 	outfit = /datum/outfit/job/ncr/f13captain
 	exp_requirements = 2000
 
-	loadout_options = list(
-	/datum/outfit/loadout/captao, //Captain armor, m16a1, deagle
-	/datum/outfit/loadout/captco, //Officer coat, R91, hunting revolver
-	)
-
 /datum/outfit/job/ncr/f13captain/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
 	..()
 	if(visualsOnly)
 		return
-	ADD_TRAIT(H, TRAIT_TECHNOPHREAK, src)
 	ADD_TRAIT(H, TRAIT_GENERIC, src)
 	ADD_TRAIT(H, TRAIT_HARD_YARDS, src)
 	ADD_TRAIT(H, TRAIT_LIFEGIVER, src)
 	ADD_TRAIT(H, TRAIT_SELF_AWARE, src)
+	ADD_TRAIT(H, TRAIT_NOSLIPALL, src)
```

Comment:
Really..? You're giving captain noslips?

Reply:
Yeah, the guy doesn't need lame heavy armor and rifles, he got enough pazazz to not fall down and look silly instead. Its a silly gimmick thing, but its fine, and when some lame trooper tries to slip him with soap the trooper going to get desert eagled in the face instead. Slipping is a very minor thing in Fallout, there are no clowns with space lube meta, its very marginal.